### DISCUSSION TITLE
How to "dx serve --no-open"?

### DISCUSSION THREAD

**Discussion by jsprog** [DioxusLabs/dioxus#2611] [Q&A]
You'd invoke `dx serve --open` if you needed to open a browser tab automatically. The `--open` argument is additive with no value at all, and It should be only enabled if stated explicitly.

The current behavior is to open by default, and I'd rather suggest introducing `no-open` instead if you're planning to keep the default behavior intact.

**Comment by xpe** [comment]
I agree. Seems to me this would be a worthwhile issue to file. (There is a "Create issue from discussion" button on the right hand side.)

**Comment by DogeDark** [comment]
This behavior should've changed in #2424
